model2vec/distill/distillation.py

```diff
@@ -276,9 +329,13 @@ def _post_process_embeddings(embeddings: np.ndarray, pca_dims: PCADimType, apply
                 logger.info(f"Explained variance ratio: {explained_variance_ratio:.3f}.")
                 logger.info(f"Explained variance: {explained_variance:.3f}.")
 
-    if apply_zipf:
-        logger.info("Applying Zipf weighting")
-        embeddings *= np.log(1 + np.arange(embeddings.shape[0]))[:, None]
+    if sif_coefficient is not None:
+        logger.info("Applying SIF and Zipf weighting")
```

Comment:
```suggestion
        logger.info("Applying SIF weighting")
```

I think we don't do zipf weighting anymore?

Reply:
It's still Zipf, because we estimate the probability through Zipf's law. I think it could be rephrased.